src/uproot/interpretation/library.py

```diff
@@ -955,130 +871,131 @@ def group(self, arrays, expression_context, how):
 
         elif uproot._util.isstr(how) or how is None:
             arrays, names = _pandas_only_series(pandas, arrays, expression_context)
```

Comment:
The `_pandas_only_series` function might also simplify because now you know that `arrays` contains _only_ Series, no DataFrames.

It still does a `_rename` thing that I think might be necessary. (I don't remember what it's for. That would be good to check: what puts a `"rename"` key in a `context`?)

Reply:
https://github.com/scikit-hep/uproot5/blob/main/src/uproot/behaviors/TBranch.py#L2900 does, I guess I'll just keep that in